README.md

```diff
@@ -36,7 +36,7 @@ A quick way to get these services running is to set up sentry, then use:
 
 Make sure you're running Python 3.7+ when setting up this environment.
 
-    mkvirtualenv snuba
+    mkvirtualenv snuba --python=$(which python3.7)
```

Comment:
This was unrelated, but is more correct. The former assumes `python` is the right version, and this is a bit more explicit. I can backout the changes from README if you'd like.